firstuseauthenticator/firstuseauthenticator.py

```diff
@@ -50,12 +57,18 @@ def _register_template_path(self):
     async def post(self):
         user = self.current_user
         new_password = self.get_body_argument('password', strip=False)
-        self.authenticator.reset_password(user.name, new_password)
+        msg = self.authenticator.reset_password(user.name, new_password)
+
+        if "success" in msg:
+            alert = "success"
```

Comment:
I'm currently not knowing as to what is contained in `msg`. Is this ok without using `.lower()` first on `msg`? I can imagine someone writing `"Successfully changed the password!"` or similar.

UPDATE: I note below that it comes from a specific message from a function specific to this very class, so it is fine!